sdk/jobs/automl-standalone-jobs/automl-image-object-detection-task-fridge-items/automl-image-object-detection-task-fridge-items-retraining-pipeline.ipynb

```diff
@@ -0,0 +1,440 @@
+{
```

Comment:
1.  Goal of this task was to convert already existed v1 notebook to v2 notebook. v1 notebook was having this functionality of retraining that if user wants to retrain whole model from scratch it can just schedule a pipeline to do so. It is basically OD pipeline + registering step in pipeline.
2. Adding support for checkpoint is a potential future task.
3. Good point -> Need to discuss.

Reply:
1. ok. So the retraining pipeline is getting the best model after training the same model from scratch multiple times and registering the best_model among all training runs.  so it is OD pipeline + register the best_model among all previous runs & the current run.
2. Makes sense